Practical5 Task- Exception Handling

Jay-Prajapat opened the pull request.

Jay-Prajapat committed:
Practical5 Task- Exception Handling

dilaara-simform commented on Practical5/Practical5/Program.cs:
```diff
@@ -0,0 +1,32 @@
+    {
+        static void Main(string[] args)
+        {
+            int[] arr = { 1, 2, 3, 4, 5 };
```
you can take dynamic input from user for array values

dilaara-simform commented on Practical5/Practical5/Program.cs:
```diff
@@ -0,0 +1,32 @@
+            {
+                try
+                {
+                    Console.WriteLine(arr[i]);
```
Print values with index position to make it more meaningful

Jay-Prajapat committed:
Updated program.cs file

-Taken dynamic input from user in array.
-Print the value in more meaningful way for array.
